Styles/DarkStyles.tsx

```diff
@@ -40,6 +38,9 @@ const styles = StyleSheet.create({
     fontSize: 24,
     fontWeight: "bold",
   },
+  iconcolor: {
```

Comment:
camelCase inconsistency but can be fixed in the next PR. Will fix conflicts myself later.